fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBReturnedRecordLimitQueryTest.java

```diff
@@ -213,17 +203,9 @@ void testComplexLimits7() throws Exception {
 
         // Scan(<,>)
         RecordQueryPlan plan = planner.plan(query);
-        if (planner instanceof RecordQueryPlanner) {
-            assertThat(plan, scan(unbounded()));
-            assertEquals(2, plan.planHash(PlanHashable.PlanHashKind.LEGACY));
-            assertEquals(-1305688464, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-            assertEquals(-1305688464, plan.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
-        } else {
-            assertThat(plan, scan(unbounded()));
-            assertEquals(2, plan.planHash(PlanHashable.PlanHashKind.LEGACY));
-            assertEquals(-690292541, plan.planHash(PlanHashable.PlanHashKind.FOR_CONTINUATION));
-            assertEquals(-690292541, plan.planHash(PlanHashable.PlanHashKind.STRUCTURAL_WITHOUT_LITERALS));
-        }
+        assertThat(plan, scan(unbounded()));
+        assertEquals(2, plan.planHash(PlanHashable.CURRENT_LEGACY));
+        assertEquals(-1686361258, plan.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
```

Comment:
This plan hash also appears to have changed

Reply:
No it hasn't this shows as changed as there used to be an `if` branching based on the used planner and it could be collapsed so I did.

Reply:
The new `CURRENT_FOR_CONTINUATION` plan hash is `-1686361258`, whereas the old `FOR_CONTINUATION` plan hash was `-1305688464` for the old planner and `-690292541` for the new planner. Meanwhile, the plan still matches `scan(unbounded())`, so it seems like it's generating the same plan but generating a different `FOR_CONTINUATION` hash, if I'm understanding all of the moving pieces currently